src/taipy/core/scenario/_scenario_manager.py

```diff
@@ -139,6 +139,12 @@ def __save_pipelines(cls, pipelines):
         for i in pipelines:
             pipeline_manager._set(i)
 
+    @classmethod
+    def _is_submittable(cls, scenario: Union[Scenario, ScenarioId]) -> bool:
+        if isinstance(scenario, str):
+            scenario = cls._get(scenario)
+        return True if isinstance(scenario, Scenario) else False
```

Comment:
```suggestion
        return isinstance(scenario, Scenario)
```
Same comment as above